### PR TITLE
Updates to swedish translations

### PR DESCRIPTION
**Pull request by MalinAhlberg**
Some proposed changes to the Swedish version.
All in all, the translations were already good! I fixed some grammatical errors, typos etc and made some sentences easier to read. I made some structural changes, like "medlem" -> "deltagare", used "ansökan" rather than "ansökningar" and removed "Bästa" (which sounds very old fashioned in Swedish ;) ).
In cases I was unsure about the translation and couldn't find any example usage on the web page, I relied on the English version.

# Definition of Done / Review checklist

## Reviewability
- [ ] link to issue
- [ ] note if PR is on top of other PR
- [ ] note if related change in rems-deploy repo
- [ ] consider adding screenshots for ease of review

## API
- [ ] API is documented and shows up in Swagger UI
- [ ] API is backwards compatible or completely new
- [ ] Events are backwards compatible

## Documentation
- [ ] update changelog if necessary
- [ ] add or update docstrings for namespaces and functions
- [ ] components are added to guide page
- [ ] documentation _at least_ for config options (i.e. docs folder)
- [ ] ADR for major architectural decisions or experiments

## Different installations
- [ ] new configuration options added to rems-deploy repository
- [ ] instance specific translations (i.e. LBR kielivara)

## Testing
- [ ] complex logic is unit tested
- [ ] valuable features are integration / browser / acceptance tested automatically

## Accessibility
- [ ] all icons have the aria-label attribute
- [ ] all fields have a label
- [ ] errors are linked to fields with aria-describedby
- [ ] contrast is checked
- [ ] conscious decision about where to move focus after an action

## Follow-up
- [ ] new tasks are created for pending or remaining tasks
- [ ] no critical TODOs left to implement

### CODE REVIEW

File: resources/translations/sv.edn [hunk 1]
```diff
@@ -406,19 +406,19 @@
...
-             :warning-about-missing-email [:<> "Du måste " [:a {:href "/settings"} "lägga till din e-postadress i dina inställningar"] ", för att kunna ta emot meddelanden om din ansökning."]}
+             :warning-about-missing-email [:<> "Du måste " [:a {:href "/settings"} "lägga till din e-postadress i dina inställningar"] ", för att kunna ta emot meddelanden om din ansökan."]}
   :table {:ascending-order "Stigande"
           :descending-order "Sjunkande"
           :all-rows-selected "Alla rader är valda"
           :no-rows-selected "Inga rader är valda"
           :some-rows-selected "Några rader är valda"
           :show-all-n-rows "Visa alla %1 rader"}
-  :unauthorized-page {:unauthorized "Ingen användningsrätt"
+  :unauthorized-page {:unauthorized "Inga användningsrättigheter"
```
Comment (opqdonut) [inline resources/translations/sv.edn:421]: I think the plural might not be right here. This message is shown when a user tries to access a page they can't see, so it's not about entitlements granted by REMS.

**Review by opqdonut — COMMENTED** [review]
Thanks, that looks nice! I have one question though.

File: resources/translations/sv.edn (hunk 1 above)
Comment (MalinAhlberg) [reply resources/translations/sv.edn:421]: Ah, I see. Using singular for this specific word is a bit weird, though. Maybe "Ingen tillgång" would be better, or even better "Du har inte tillgång till denna sida".

Comment (opqdonut) [reply resources/translations/sv.edn:421]: `:unauthorized` is the title of the page so "Ingen tillgång" sounds great for that. `:you-are-forbidden` could be the longer "Du har inte tillgång ..." text.

Comment (MalinAhlberg) [reply resources/translations/sv.edn]: Just pushed a commit with that :+1:

**Review by opqdonut — APPROVED** [review]

**Comment by opqdonut** [comment]
we're discussing whether we want to merge this before or after our next release on thursday

**Pull request merged**